packages/edit-site/src/components/header/theme-udpdater/index.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * External dependencies
+ */
+import { filter, some } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { store as noticesStore } from '@wordpress/notices';
+import {
+	__experimentalUseEntityRecords as useEntityRecords,
+	store as coreStore,
+} from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import { store as editSiteStore } from '../../../store';
+import isTemplateRevertable from '../../../utils/is-template-revertable';
+
+export default function ThemeUpdater() {
+	const [ canSave, setCanSave ] = useState( false );
+	useEffect( async () => {
+		const userProps = await apiFetch( {
+			path: 'wp/v2/users/me?context=edit',
+		} );
+
+		setCanSave( userProps?.capabilities?.edit_theme_options );
```

Comment:
I think this should be `edit_themes` not `edit_theme_options`.

Reply:
Solved in https://github.com/WordPress/gutenberg/pull/40385/commits/7b0314c43e8e6a364ed26e9d6dbfc692ced9c076